### PR TITLE
Add shopping files

### PR DESCRIPTION
**Pull request by rascalking**
Configured to allow shopping for Fall 2015 term.

### CODE REVIEW

File: css/shop.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/* School CSS */
```
Comment (sapnamysore) [inline css/shop.css:1]: We don't need placeholder for School css anymore, right?

File: js/shop.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
+// allowed_terms is a whitelist of Canvas enrollment_term_ids where shopping is allowed
+// NOTE - the term ids in allowed_terms must be strings, not ints
+var allowed_terms = ["69"];
```
Comment (sapnamysore) [inline js/shop.js:3]: Same question as in the colgsas PR. Is 69 the right term to be checked in or should it be blank?

File: js/shop.js [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+var shopping_tool_url = "https://icommons-tools.tlt.harvard.edu/shopping";    // the url of the shopping tool
+
```
Comment (sapnamysore) [inline js/shop.js:5]: More a question for devops I guess. @cmurtaugh/@ekiczek:  wouldn't the shopping_tool_url change  with the tools moving to AWS?

**Comment by sapnamysore** [comment]
Just a clarification on term ids and the shopping url.  Other than that :+1:

File: js/shop.js (hunk 3 above)
Comment (cmurtaugh) [inline js/shop.js:5]: Hi -- that is the new AWS-hosted production url.

Comment (sapnamysore) [inline js/shop.js:5]: Perfect :-)

File: css/shop.css (hunk 1 above)
Comment (rascalking) [inline css/shop.css:1]: Nope, but I didn't want to mess with this without a reason...it's basically just copied over from icommons_tools.

**Pull request merged**